src/githubHelper.ts

```diff
@@ -1413,6 +1431,66 @@ export class GithubHelper {
     return `Commented on [${ref}](${repoLink}/compare/${base_sha}..${head_sha}${slug})\n\n`;
   }
 
+  async addIssueInformation(issue: GitLabIssue | GitLabMergeRequest, description: string): Promise<string> {
+    let bodyConverted = description;
+
+    let mrAssignees = issue.assignees.map(a => a.username);
+    if (settings.usermap) {
+      let assignees = [];
+      for (let assignee of mrAssignees) {
+        let githubUser = settings.usermap[assignee as string];
+        if (githubUser) {
+          githubUser = '@' + githubUser;
+        } else {
+          githubUser = assignee as string;
+        }
+        assignees.push(githubUser);
+      }
+
+      if (assignees.length > 0) {
+        bodyConverted += '\n\n**Assignees:** ' + assignees.join(', ');
+      }
+    }
```

Comment:
Could you refactor this into a function which gets called with assignees, reviewers and approvals as it seems they are nearly the same

Reply:
Good point! Moved it into a util function. If you have an idea for a better name for the function, let me know.